pkg/converter/muxer.go

```diff
@@ -127,3 +128,24 @@ func (m *Muxer) ToUnstructured() (*unstructured.Unstructured, error) {
 	}
 	return d, nil
 }
+
+// ToObject converts input data to the correct structured type.
+func (m *Muxer) ToObject() (runtime.Object, error) {
+	// TODO: I think we can just use a Scheme for this
```

Comment:
yeah, you sure can. I was thinking about adding this but I haven't gotten around to it yet. There's a vestigial scheme that's still living in https://github.com/GoogleCloudPlatform/k8s-cluster-bundle/blob/master/pkg/scheme/scheme.go that I was thinking about reintroducing with this purpose.